src/utils/useTxMulti/useTxMulti.ts

```diff
@@ -30,6 +34,28 @@ export type TUseTxMulti = () => {
 export const useTxMulti: TUseTxMulti = () => {
   const [state, dispatch] = useReducer(TxMultiReducer, initialState, identity, 'TxMulti');
   const getState = () => state;
+  const { accounts } = useContext(StoreContext);
+  const { addNewTxToAccount } = useContext(AccountContext);
+  const { assets } = useAssets();
+  const { getNetworkByChainId } = useNetworks();
+  const { account } = state;
+
+  const currentTx: TxParcel = view(lensIndex(state._currentTxIdx), state.transactions);
+
+  useEffect(() => {
+    if (
+      account &&
+      currentTx &&
+      currentTx.txResponse &&
+      currentTx.txHash &&
+      currentTx.status === ITxStatus.BROADCASTED
+    ) {
+      const network = getNetworkByChainId(currentTx.txRaw.chainId)!;
```

Comment:
Isn't `network` already available in `state`?

Reply:
I hadn't spotted that, but yeah!